### ISSUE TITLE
Issues with version 5.1 pom

### ISSUE THREAD

**Issue by jesjos** [zalando/spring-cloud-config-aws-kms#100]
I've been struggling to upgrade to version 5.1 of this package. When compiling my maven project I get an error like this:

```
Failed to read artifact descriptor for org.zalando:spring-cloud-config-aws-kms:jar:5.1: Could not find artifact org.zalando:spring-cloud-config-aws-kms-parent:pom:5.1
```

In maven central, the POM looks like this: https://repo1.maven.org/maven2/org/zalando/spring-cloud-config-aws-kms/5.1/spring-cloud-config-aws-kms-5.1.pom

There's a `parent` tag pointing to a project that's not present in maven central.

Is this expected? Is there some workaround? Or is there something amiss with the release?

Thanks for making a great lib!

**Comment by harti2006** [comment]
Ah, I probably shouldn't have excluded the parent pom from being uploaded to maven central. I wanted to keep things clean, but was too eager. Let me try to fix this and release a new version 5.1.1...

**Comment by harti2006** [comment]
5.1.1 was released. However it could take some time until the new artifact `spring-cloud-config-aws-kms-parent` appears. I will continue testing...

**Comment by jesjos** [comment]
Seems this was fixed in 5.1.2 but 5.1.1 is giving me errors.
